src/edit.test.js

```diff
@@ -0,0 +1,19 @@
+const editTask = require('./edit.js');
+
+const previousTask = [
+  { description: 'Task1', completed: false, index: 1 },
+  { description: 'Task2', completed: false, index: 2 },
+  { description: 'Task3', completed: false, index: 3 },
+];
+
+const newTask = [
+  { description: 'Task1', completed: false, index: 1 },
+  { description: 'Update GitHub Profile', completed: false, index: 2 },
+  { description: 'Task3', completed: false, index: 3 },
+];
+
+test('Update ToDo item', () => {
+  expect(
+    editTask(previousTask, 2 - 1, 'Update GitHub Profile'),
+  ).toEqual(newTask);
+});
```

Comment:
- [ ] According to the project requirement, please kindly group your test using the `describe()` method. **Optionally**, you can add one or more tests in order to bring out the essence of the `describe()` method. A describe block is used for organizing test cases in logical groups of tests. ℹ️ 

**> Note, I commend how you implemented the test cases for `localstorage.test.js`. 👍**